pkg/controllers/managementuser/alert/initdata.go

```diff
@@ -284,7 +284,7 @@ var entries = []entry{
 					},
 					MetricRule: &v32.MetricRule{
 						Description:    "The cpu load is higher than 100",
-						Expression:     `sum(node_load1) by (node)  / sum(machine_cpu_cores) by (node) * 100`,
+						Expression:     `sum(node_load1) by (node)  / sum(kube_node_status_capacity_cpu_cores) by (node) * 100`,
```

Comment:
This is a deprecated metric that comes from `kube-state-metrics`. It will officially be removed in v2.x based on https://github.com/kubernetes/kube-state-metrics/issues/974.

However, this should be okay since:
- The change only impacts Alerting V1, not Alerting V2. Alerting V2 does not use MetricRules afaik (@codyrancher we aren't using this for https://github.com/rancher/dashboard/issues/812 right?)
- Alerting V2 is still on version 1.9.x of kube-state-metrics and is deprecated, so that is unlikely to change.

@guangbochen I'm adding you as a reviewer to also get a sign-off since this is a Monitoring / Alerting V1 issue. I want to confirm that using this metric is okay and that we have no plans of upgrading the `kube-state-metrics` version of Monitoring / Alerting V1.

Reply:
this is LGTM since for v1 monitoring the latest v1 monitoring chart is using v1.9.7 `kube-state-metrics` and the metric is supported.
https://github.com/rancher/system-charts/blob/dev-v2.5/charts/rancher-monitoring/v0.2.0/values.yaml#L187

Reply:
@dnoland1, based on https://github.com/kubernetes/kubernetes/issues/95204#issuecomment-719445180, should the correct, non-deprecated call be the following?
```suggestion
						Expression:     `sum(node_load1) by (node)  / sum(kube_node_status_capacity{resource="cpu"}) by (node) * 100`,
```